### PR TITLE
feat: jump to bottom on message send

### PR DESCRIPTION
**Pull request by suprko**
#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/6873/desktop-autoscroll-when-sending-a-new-message-to-the-chat-yourself

#### Testing instructions  
Sending a message should cause the message list to scroll to the bottom.

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

**Comment by suprko** [comment]
Pretty simple but I don't know if this is the best way to implement this feature...?

**Comment by anri-asaturov** [comment]
here

**Comment by anri-asaturov** [comment]
Your way was not the best bcs 
1. it only worked when sending messages but not ACK or files
2. it used 'jump to bottom' that works a bit differently, sometimes it can reset the chat, which send will do anyway, and then there will be 2 resets

File: src/ui/chat/ChatView.jsx [hunk 1]
```diff
@@ -71,32 +77,35 @@ class ChatView extends React.Component {
      * @param {Object} richText A ProseMirror document tree, in JSON.
      * @param {string} legacyText The rendered HTML of the rich text, for back-compat with older clients
      */
-    sendRichTextMessage(richText, legacyText) {
+    @action.bound sendRichTextMessage(richText, legacyText) {
         try {
+            this.scrollToBottom();
```
Comment (suprko) [inline src/ui/chat/ChatView.jsx]: should use `jumpToBottom` here no? if we're scrolled up high enough that the newest chats have unloaded, we need `chat.reset` in order to actually go to the newest chat

Comment (anri-asaturov) [reply src/ui/chat/ChatView.jsx]: send() in sdk does the reset if canGoDown

**Pull request merged**